tests/integration/crawlerConductor.test.js

```diff
@@ -5,7 +5,7 @@ const {createCollector} = require('../../helpers/collectorsList');
 const testURLs = [
```

Comment:
changes to integration tests are unrelated - pages used for integration testing changed. BTW I plan to use our own privacy test page for fingerprint testing instad of fingerprintjs, but that's outside of the scope of this PR.